site/blog/airship-update-february-2020.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Airship Update - February 2020
+author: alex-hughes
+date: 2020-02-12T12:00:00.000Z
+category: Airship 2.0
+---
+
+What follows is the Airship Community Update for the month of February 2020, brought to you by the Airship Technical Committee.<!-- more -->
+
+## **SPOTLIGHT ON ....**
+
+- The Airship Technical and Working Committees each had special elections in January. We welcome
+    - Andrew Leasck (AT&T) to the Technical Committee
+    - Ian Pittwood (Accenture) to the Working Committee
+- Airship's year in review was featured in the OpenStack Foundation Annual Report. For more information
+  see this [blog post](https://www.airshipit.org/blog/a-year-in-review-getting-confirmed-and-looking-ahead-to-2-0.html).
+
+<br>
+
+## **AIRSHIP BLOG**
+
+The Airship blog is a great way to keep up with what's going on in the community. The Airship community publishes
+[*blog posts*](https://www.airshipit.org/blog/) regularly, including the recent Airship 2.0 Blog series. These blog
+posts introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail the evolution of each
+component. The first six Airship 2.0 Blog posts are already available:
+
+- [*Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+- [*Airship Blog Series 2 - An Educated Evolution*](
+   https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+- [*Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+   https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
+- [*Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+   https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+- [*Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+- [*Airship Blog Series 6 - Armada Growing Pains*](
+   https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
+
+<br>
+
+## **AIRSHIP 2: THE SEQUEL**
+
+Recently Ryan van Wyk narrated a great, short, video discussing Airship - and the evolution to Airship2.  It's a must
+see for anyone interested in Airship.
+
+[![](/images/airship2-video-thumbnail.png)](https://www.youtube.com/watch?v=xt400lkgvTA)
+
+<br>
+
+## **JIRA MIGRATION TO GITHUB ISSUES**
+The Airship Community is embracing [GitHub Issues](https://github.com/airshipit/airshipctl/issues) for tracking Airship
```

Comment:
There's more in the Airship space besides airshipctl. We can include it as an example, but we might want to broaden the general scope of this message since all projects are switching to GitHub Issues eventually.